src/meta/src/barrier/recovery.rs

```diff
@@ -131,6 +146,11 @@ impl GlobalBarrierManager {
         let state = tokio_retry::Retry::spawn(retry_strategy, || {
             async {
                 let recovery_result: MetaResult<_> = try {
+                    // Mview progress needs to be recovered.
+                    tracing::info!("recovering mview progress");
+                    self.recover_mview_progress().await?;
```

Comment:
Can we just call `recover_mview_progress` once for each recovery rather than each retry inside the recover, just like the cleaning logic?

Reply:
Makes sense. Let me change it.